test/components/elements/ErrorMessageTest.js

```diff
@@ -1,25 +1,54 @@
 'use strict'
 import { expect } from 'chai'
+import moment from 'moment'
 import createComponent from 'test/helpers/shallowRenderHelper'
 import ErrorMessage from 'components/elements/ErrorMessage'
+import { Card, CardHeader } from 'material-ui/Card'
 import ErrorIcon from 'material-ui/svg-icons/alert/error'
 import { RetryButton } from 'components/elements/Buttons'
 
 describe('ErrorMessage', function () {
-  it('Should render an ErrorIcon icon and a span with the given message when called with just a message', function () {
+  describe('when given only a message', function() {
     const testObject = ErrorMessage('TestMessage')
 
-    expect(testObject.type).to.equal('div')
-    expect(testObject.props.children[0].type).to.equal(ErrorIcon)
-    expect(testObject.props.children[1].type).to.equal('span')
-    expect(testObject.props.children[1].props.children).to.equal('TestMessage')
-    expect(testObject.props.children[2]).to.be.null
+    it('Should render a Card with an ErrorIcon icon and a CardHeader with the given message', function () {
+      expect(testObject.type).to.equal('div')
+      expect(testObject.props.children.type).to.equal(Card)
+      expect(testObject.props.children.props.children[0].type).to.equal(ErrorIcon)
+      expect(testObject.props.children.props.children[1].type).to.equal(CardHeader)
+      expect(testObject.props.children.props.children[1].props.title).to.equal('TestMessage')
+    })
+
+    it("should render a card with a CardHeader with the subtitle 'Time unknown!", function () {
+      expect(testObject.props.children.props.children[1].props.subtitle).to.equal('Time unknown!')
+    })
   })
 
-  it('Should additionally render a retry button when provided an action for retry', function () {
+  describe('when given a message and provided an action for retry', function() {
     const testObject = ErrorMessage('TestMessage', 'TestAction')
 
-    expect(testObject.props.children[2].type).to.equal(RetryButton)
-    expect(testObject.props.children[2].props.actionForRetry).to.equal('TestAction')
+    it('Should additionally render a retry button', function () {
+      expect(testObject.props.children.props.children[2].type).to.equal(RetryButton)
+      expect(testObject.props.children.props.children[2].props.actionForRetry).to.equal('TestAction')
+    })
+  })
+
+  describe('when given a message, an action, and a time', function () {
```

Comment:
It might be best to test in full isolation, giving just a time and expecting the subtitle, but not testing any other aspect. That's all that's covered by your expect statement, so including the message and action doesn't seem needed. You could just pass in null for the first two arguments and the rest of the test remains the same. (and the describe would state just 'when given a time', making it more clear precisely what's being tested).

Same for the next describe statement; a test of just the background color with null message and action should suffice.